src/libs/actions/Policy.js

```diff
@@ -718,7 +719,23 @@ function generateDefaultWorkspaceName(email, policyList) {
         defaultWorkspaceName = 'My Group Workspace';
     }
 
-    return hasPolicies ? `${defaultWorkspaceName} ${policyListLength + 1}` : defaultWorkspaceName;
+    if (defaultWorkspaceName === '') {
+        return defaultWorkspaceName;
+    }
+    console.log('this is all the polices ', getAllPolicies());
+
```

Comment:
let's check the length of all policies first, if there isn't anything just return the policy name before doing the foreach loop